webpack.conf.js

```diff
@@ -1,19 +1,40 @@
+const path = require('path');
+
 module.exports = {
-    devtool: 'inline-source-map',
+    mode: 'production',
+    entry: './src/main.js',
+    output: {
+        path: path.join(__dirname, 'dist'),
+        filename: 'stencil-utils.min.js',
+        library: ['stencilUtils'],
+    },
+    optimization: {
+        minimize: true,
+    },
+    devtool: false,
```

Comment:
Is there a reason why we are changing this to false ?

Reply:
devtool is used only in mode: 'development'. We are using 'production' build, so we don't need to specify it